src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -236,18 +266,25 @@ export class CodeActionMenu extends Disposable implements IEditorContribution {
 		}, [this.listRenderer], { keyboardSupport: false }
 		);
 
+		renderDisposables.add(this.codeActionList.value.onMouseOver(e => this._onListHover(e)));
+		renderDisposables.add(this.codeActionList.value.onDidChangeFocus(e => this.codeActionList.value?.domFocus()));
 		renderDisposables.add(this.codeActionList.value.onDidChangeSelection(e => this._onListSelection(e)));
 		renderDisposables.add(this._editor.onDidLayoutChange(e => this.hideCodeActionWidget()));
 
 
 		// Populating the list widget and tracking enabled options.
 		inputArray.forEach((item, index) => {
 			const currIsSeparator = item.class === 'separator';
+			let isDocumentation = false;
+			if (item instanceof CodeActionAction) {
+				isDocumentation = item.action.kind === '_documentation';
```

Comment:
This should also be a constant instead of a string literal

Reply:
The `_documentation` part? ATM isDocumentation is just a boolean and item.action.kind is a string (checking against if it is actually a documentation type or not)

Reply:
Yes `_documentation` should be constant so that it's clear you're using this string to pass info between places in the code